llvm-bazel/llvm-project-overlay/llvm/unittests/BUILD

```
@@ -289,6 +289,7 @@ cc_test(
     srcs = glob(["FrontEnd/*.cpp"]),
```

Let's include fixing this glob as part of this PR?

```suggestion
    srcs = glob(["Frontend/*.cpp"]),
```